core/deployment/src/main/java/io/quarkus/deployment/builditem/JniBuildItem.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.deployment.builditem;
+
+import java.util.List;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class JniBuildItem extends MultiBuildItem {
+
+    private final List<String> libraryPaths;
+
+    public JniBuildItem(List<String> libraryPaths) {
```

Comment:
I think we also need an empty constructor as you might want to enable JNI even without any library path (this is the case for the Tika extension for instance).